usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -169,10 +168,8 @@ def get_download_ids(keyword, field, size=10000):
 
     max_iterations = 10
     total = get_total_results(keyword, '*', max_iterations)
-    if not total:
+    if total is not None:
         logger.error('Error retrieving total results. Max number of attempts reached')
-        return None
-
     n_iter = min(max(1, total // size), n_iter)
```

Comment:
What happens on this line when total is None?

Reply:
It breaks if total is None. Still testing.

Reply:
This has been resolved with a yield, then return. it's been tested.